app/script/event/EventRepository.js

```diff
@@ -638,7 +638,13 @@ z.event.EventRepository = class EventRepository {
     return this.conversationService.load_event_from_db(conversationId, eventId).then(storedEvent => {
       if (storedEvent) {
         const {data: mappedData, from: mappedFrom, type: mappedType, time: mappedTime} = event;
-        const {data: storedData, from: storedFrom, type: storedType, time: storedTime} = storedEvent;
+        const {
+          data: storedData,
+          from: storedFrom,
+          type: storedType,
+          time: storedTime,
+          primary_key: storedPrimaryKey,
+        } = storedEvent;
```

Comment:
We only use the primary key in the case all previous checks passed. Let's simplify the destructuring, remote the primary_key entry here so it fits into a single line and explicitly set it in L685